src/content/_posts/2020-10-08-working-with-the-community-and-launch-of-the-new-design.md

```diff
@@ -0,0 +1,85 @@
+but I think it's time for getting your feedback!
+
+But first let me talk real quick a little bit about the community.
+
+## Community
+
+Without all the private feedback I've got over the last couple of weeks, this would just not been possible.
+
+That's why I **really love** working with the community.
+
+> Some special thanks to [Paramethod](https://github.com/Paramethod), [hazard999](https://github.com/hazard999), [egarim](https://github.com/egarim)  
+> and of course our newest MVP [jjcolumb](https://github.com/jjcolumb)! ????
+
+But what does it mean to be part of a community?
+
+I wrote about this [in the past](2019/12/05/the-third-annual-csharp-advent-pretzel-journey.html#community), but just a quick recap what it means working as an MVP in an open community and what *not*.
+
+Back in the days I got voted as an MVP (I can't remember exactly when, @Dennis if you know, let me know in the comments below) I mainly did read the [supportcenter](https://supportcenter.devexpress.com/) tickets related to [XAF](tags/XAF.html) and [XPO](tags/XPO.html) and try to learn more about the framework and eco system and tried to answer questions I know an solution to, as well as getting inspiration how to make our team more productive.
+
+Also I tried to make the **hard** things *easier* by throwing out some [open source stuff](https://github.com/biohazard999?tab=repositories&q=&type=source&language=).  
+To mention a few with more traction:
+
+* [DXNugetPackageBuilder](https://github.com/biohazard999/DXNugetPackageBuilder)
+* [FluentModelBuilder](https://github.com/biohazard999/Para.FluentModelBuilder)
+* [DXImageAssemblyImporter](https://github.com/biohazard999/DXImageAssemblyImporter)
+* [WindowsIntegration](https://github.com/biohazard999/Xpand.ExpressApp.Win.Para.WindowsIntegration) (this was the first one)
+
+Some of them are obsolete now (`DXNugetPackageBuilder` is not longer needed, but for legacy apps it's still valuable).
+
+I put a lot of effort and time into this in the past and I am always happy if I put something out there that helped *somebody else* and I will continue to do so.
+
+This is the **giving** back to the community part.
+But a community is not only about *giving* it's also about **taking**.
+
+I do it every day. Using open source, reading blog-posts, articles, docs, samples, reaching out to people helping me to achieve **my goal**.
+
+People are investing hundreds of thousands of hours into problem solving online.
+
+* Write a blog post about a problem so I have a space to look at if I ever encounter this again
+* Write a tutorial / in depth for reference
+* Write open source software
+* Contribute by
+  * Comment
+  * Fix spelling
+  * Provide documentation
+  * Testing
+  * Filing bugs/issues/ideas
+* Provide funding
+
+The last point is interesting, cause we all know that we need money for living, but let me focus know on parts that I don't think are part of an open community:
```

Comment:
```suggestion
The last point is interesting, because we all know that we need money for living, but let me now focus on parts that I don't think are part of an open community:
```